airflow/api_connexion/schemas/task_instance_schema.py

```diff
@@ -158,7 +158,9 @@ class SetTaskInstanceStateFormSchema(Schema):
     include_downstream = fields.Boolean(required=True)
     include_future = fields.Boolean(required=True)
     include_past = fields.Boolean(required=True)
-    new_state = TaskInstanceStateField(required=True, validate=validate.OneOf([State.SUCCESS, State.FAILED]))
+    new_state = TaskInstanceStateField(
+        required=True, validate=validate.OneOf([State.SUCCESS, State.FAILED, State.SKIPPED])
```

Comment:
Let’s change these to use TaskInstanceState instead (also the one below)

Reply:
> Let’s change these to use TaskInstanceState instead (also the one below)

Good point, I didn't dare but you're right, just did it.